nuxeo-data-table/iron-data-table.html

```diff
@@ -472,15 +472,22 @@ <h2>[[i18n('command.add')]]</h2>
               type: Boolean,
               value: false,
             },
+            /**
+             * Extended Nuxeo-Data-Table-Checkox tag name
+             */
+            _TagName: {
+              type: String,
+              value: 'NUXEO-DATA-TABLE-CHECKBOX',
+            },
           };
         }
 
-        constructor() {
+        constructor(columnTagName = 'NUXEO-DATA-TABLE-COLUMN') {
```

Comment:
I guess you must assign somewhere in the constructor `this._tagName = columnTagName` ...

Reply:
_tagName is for the checkbox (i will rename it to _checkboxTagName to avoid confusions=, columnTagName is used only locally in the data-table constructor.